### PR TITLE
AnnotationBear: Remove 'language_family'

### PR DESCRIPTION
**Pull request by abhsag24**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Use >= for development versions so that source builds always work
-coala>=0.7.0.dev20160529235806
+coala>=0.7.0.dev20160602092230
```
Comment (tushar-rishav) [inline requirements.txt]: Unrelated changes. Right?
@abhsag24

Comment (tushar-rishav) [inline requirements.txt]: Oh. Ignore ^

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-bears/pull/480/commits/d32dfd1387e3632339310a5f0c9595b953096b3a https://github.com/coala-analyzer/coala-bears/pull/480/commits/b44a25d9acbca454b7ca080854a167128091579d

**Comment by sils1297** [comment]
@rultor merge
